src/Directory.Build.props

```diff
@@ -1,3 +1,5 @@
 <Project>
   <Import Project="..\Directory.Build.props" />
+  <!-- Turn off end of life target framework checks as we intentionally build older .NETCoreApp configurations. -->
+  <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
a. This needs to be in a `<PropertyGroup>`
b. doesn't this need to go higher since there are .csprojs in the `test` folder?

Reply:
Yup yup, I was just testing :) Doing those changes locally